.vscode/launch.json

```diff
@@ -11,7 +11,7 @@
 			"stopOnEntry": false,
 			"sourceMaps": true,
 			"outFiles": ["${workspaceRoot}/out/**/*.js"],
-			"preLaunchTask": "npm"
+			"preLaunchTask": "npm: compile"
```

Comment:
The `tasks.json` file is no longer neccessary now that `npm` scripts are auto-detected, so we can just simply specify the `npm` script we want to run directly from here (I love this capability!).